### ISSUE TITLE
3.1.0 don't show done/next button in only one step

### ISSUE THREAD

**Issue by DigitalAndSEO** [usablica/intro.js#1101]
I have tried all possible combinations of "show buttons", "nextToDone", etc ...
And when there is only one step, it does not show any type of button.

```
var recuerdaAgentes = introJs();
    recuerdaAgentes.setOptions({
        steps: [
        {
        element: '#agente1',
        intro: 'Sr. PRIMER APELLIDO, le recordamos que nuestro personal disponible puede <b>atenderle en vivo</b> cuando lo necesite durante el horario de atención:<br>9:00h - 19:00h'
        }
        ],
        nextLabel: 'De acuerdo, gracias',
        doneLabel: 'De acuerdo, gracias',
        overlayOpacity: .8,
        exitOnOverlayClick: false,
        exitOnEsc: false,
        keyboardNavigation: false,
        nextToDone: true,
        showButtons: true,
        showBullets: false,
        showProgress: false,
        showStepNumbers: false,
        disableInteraction: true

    });
    recuerdaAgentes.start();
```

HTML RESULT: https://jmp.sh/lQNnE2V
SCREEN RESULT: https://jmp.sh/vXeDyX1

**Comment by DigitalAndSEO** [comment]
Could you give me some kind of solution? I've had the case open for 12 days and I have no way of showing the DONE BUTTON.
Thank you.

**Comment by ZsharE** [comment]
Same problem here.
You could try using this code in your console on https://introjs.com:
```
introJs().setOptions({
	steps: [{
		intro: "Some test message here."
	}]
}).start();
```

**Comment by DigitalAndSEO** [comment]
Exactly @ZsharE . Even on your website this happens.
If it is not solved soon I will have to use another type of js or downgrade to version 2.9.3.
I hope you can give us a quick solution because it worked for me with version 2.9.3, but I wanted to do an update and I changed a lot of code to adapt it to version 3.1.0. :(

**Comment by ZsharE** [comment]
https://github.com/usablica/intro.js/blob/master/src/core/showElement.js#L380

Should be:
`if (this._introItems.length >= 1) { ... }`

**Comment by DigitalAndSEO** [comment]
> Should be: `if (this._introItems.length >= 1) { ... }`

It works perfectly!
If the general file "intro.js" is used, the modification is made in line 2867.

**Comment by afshinm** [comment]
Sorry about that that folks, could you please test this PR https://github.com/usablica/intro.js/pull/1118 -- happy to release a version as soon as that's approved.

**Comment by afshinm** [comment]
cc @DigitalAndSEO and @ZsharE

**Comment by afshinm** [comment]
there is also a Netlify dev preview https://deploy-preview-1118--introjs-repo.netlify.app/example/ for that PR

**Comment by DigitalAndSEO** [comment]
Test & Work

**Comment by afshinm** [comment]
@DigitalAndSEO I'm guessing that's because you've changed "line 2867" to `if (this._introItems.length >= 1) { ... }`. 

> Should be: if (this._introItems.length >= 1) { ... }

I cannot reproduce that bug.

**Comment by DigitalAndSEO** [comment]
That's. If instead of "> =" I leave the original ">" it doesn't work, because there is only one step

**Comment by afshinm** [comment]
@DigitalAndSEO not sure if I follow. if you open https://deploy-preview-1118--introjs-repo.netlify.app/example/hello-world/index.html and run the following code:

```javascript
introJs().setOptions({
    steps: [{
        intro: "Some test message here."
    }]
}).start();
```

in your Dev console, you'll only see the "Done" button with no "Previous" button because there is only one step.

**Comment by ZsharE** [comment]
Tested, works fine now. 👍

**Comment by DigitalAndSEO** [comment]
@afshinm I have just verified that if only one step is used but without assigning an element to it, the overlay does not appear, so the message does not attract attention.
I think it should show anyway, whether it has an element assigned or not.

**Comment by DigitalAndSEO** [comment]
Will you upload it to CDNs as a new 3.1.1 update?

**Comment by afshinm** [comment]
Just published v3.2.0 https://github.com/usablica/intro.js/releases/tag/v3.2.0

**Comment by DigitalAndSEO** [comment]
> Just published v3.2.0 https://github.com/usablica/intro.js/releases/tag/v3.2.0

You will also publish the changes in https://cdnjs.com/libraries/intro.js ?

**Comment by afshinm** [comment]
CDNs are update automatically from our NPM package, see https://www.jsdelivr.com/package/npm/intro.js

not sure how long it takes to see the changes on cdnjs.
